build/transpile.js

```diff
@@ -1690,7 +1690,7 @@ class Transpiler {
                 'OrderType': 'string',
                 'OrderSide': 'string',
                 'Dictionary<any>': 'array',
-                'Dict': 'array',
+                'Dict': '?array',
```

Comment:
I'm not sure how this is related to safeNumber because safe number can accept either a list or a dict, so the TS type won't be `Dict`